### PR TITLE
Usecase 1.x

### PR DESCRIPTION
**Pull request by hsorby**
Addresses issue #57

### CODE REVIEW

**Comment by hsorby** [comment]
Ok this is all the tests written to satisfy the use cases, mentioning people so that they get an email. @nickerso, @agarny.

**Comment by nickerso** [comment]
The tests all look good to me.

File: tests/model/units_import.cpp [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+               "</units>"
+            "</model>";
+
+    libcellml::Model m;
+    libcellml::ImportPtr imp = std::make_shared<libcellml::Import>();
+    imp->setSource("some-other-model.xml");
+
+    libcellml::UnitsPtr importedUnits = std::make_shared<libcellml::Units>();
+    importedUnits->setName("units_in_this_model");
+    importedUnits->setSourceUnits(imp, "units_in_this_model");
```
Comment (agarny) [inline tests/model/units_import.cpp]: Shouldn't it be `units_in_that_model`?

**Comment by agarny** [comment]
Apart from the minor typo above, it also looks good to me.

**Comment by nickerso** [comment]
Looks like @hsorby touched up the typo I completely missed but thankfully @agarny looks a lot closer than I obviously do! Seems to be all good to merge and all tests are passing.

**Pull request merged**